security-self-review.md

```diff
@@ -0,0 +1,81 @@
+# NativeIO Security Self-Review
+
+https://www.w3.org/TR/2020/NOTE-security-privacy-questionnaire-20200508/
+
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+Since this feature is a storage API, it does not expose any information that is
+not first created by the origin.
+
+## 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes, we only expose data that was persisted by the origin.
+
+## 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+This specification does not expose any personal information.
+
+## 2.4. How does this specification deal with sensitive information?
+
+This specification does not deal with sensitive information.
+
+## 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Yes, this specification introduces a new persistent storage mechanism where
+state may be kept. The risk of tracking must be mitigated by following similar
+strategies used in other storage APIs e.g. user agents should allow users to
+wipe stored data, an origin may only access files that are owned by it, etc.
+
+## 2.6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+No information from the underlying platform is exposed. However care must be
+taken during implementation, such as not to expose operating-system-specific
+behavior through the way file operations are executed.
+
+## 2.7. Does this specification allow an origin access to sensors on a user’s device
+
+This specification does not allow access to sensors.
+
+## 2.8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+The data exposed is only the one that was previously created by the origin. This
+is similar to other storage APIs
+
+## 2.9. Does this specification enable new script execution/loading mechanisms?
+
+This specification does not enable new script execution/loading mechanisms.
```

Comment:
Nit: Below, you omitted the newline before the next heading

Reply:
Done!